app/javascript/controllers/selection_controller.js

```diff
@@ -49,4 +59,10 @@ export default class extends Controller {
       JSON.stringify([...storageValue])
     );
   }
+
+  #updateActinLinks(count) {
```

Comment:
Can you add a comment to why the `#` is infront of the function name here. Also there's a typo in the function name

Reply:
Added in 71d0b560f7ff2efa4c9fa50b28ecf755469fe42a